analyzer/codechecker_analyzer/analyzer_context.py

```diff
@@ -376,6 +486,26 @@ def checkers_guideline_map_file(self):
         return os.path.join(self._package_root, 'config',
                             'checker_guideline_map.json')
 
+    @property
+    def checker_profile_map_file(self):
+        """
+        Returns the path of checker-profile mapping config file. This file
+        may come from a custom location provided by CC_PROFILE_MAP_FILE
+        environment variable.
+        """
+        # Get profile map file from the environment.
+        if 'CC_PROFILE_MAP_FILE' in os.environ:
```

Comment:
Please document this environment variable here and update the documentation after it: https://github.com/Ericsson/codechecker/blob/0a7617b897f55827a162ab76cecfba2a370a3e76/analyzer/codechecker_analyzer/cmd/analyze.py#L51